client/landing/jetpack-cloud/components/server-credentials-wizard-dialog/index.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import classnames from 'classnames';
+import { Dialog } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import Gridicon from 'components/gridicon';
+import ServerCredentialsForm from 'landing/jetpack-cloud/components/server-credentials-form';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	onCloseDialog: Function;
+	showDialog: boolean;
+	siteId: number;
+	showServerCredentialStep: boolean;
+	children: React.ReactNode;
+	buttons?: React.ReactNode;
+	baseDialogClassName?: string;
+	title: i18nCalypso.TranslateResult;
+	titleClassName?: string;
+}
+
+const ServerCredentialsWizardDialog = ( {
+	onCloseDialog,
+	showDialog,
+	siteId,
+	showServerCredentialStep,
+	buttons,
+	title,
+	titleClassName,
+	baseDialogClassName,
+	children,
+}: Props ) => {
+	return (
+		<Dialog
+			additionalClassNames={ classnames( 'server-credentials-wizard-dialog', baseDialogClassName ) }
+			isVisible={ showDialog }
+			buttons={ showServerCredentialStep ? undefined : buttons }
+			onClose={ onCloseDialog }
+		>
+			<h1 className={ classnames( 'server-credentials-wizard-dialog__header', titleClassName ) }>
+				{ title }
+			</h1>
+			{ showServerCredentialStep && (
+				<>
+					<h3 className="server-credentials-wizard-dialog__subheader">
+						{ translate( 'You have selected to fix all discovered threats' ) }
+					</h3>
+					<div className="server-credentials-wizard-dialog__warning">
+						<Gridicon
+							className="server-credentials-wizard-dialog__icon server-credentials-wizard-dialog__icon--warning"
+							icon="info"
+							size={ 36 }
+						/>
+						<p className="server-credentials-wizard-dialog__warning-message">
+							{ translate(
+								"Jetpack is unable to auto fix these threats as we currently do not have access to your website's server. Please supply your SFTP/SSH credentials to enable auto fixing. Alternatively, you will need go back and {{strong}}fix the threats manually{{/strong}}.",
+								{
+									components: {
+										strong: <strong />,
+									},
+								}
+							) }
+							<br />
+						</p>
+					</div>
+
+					<ExternalLink
+						className="server-credentials-wizard-dialog__link-external"
+						icon
+						href="https://jetpack.com/support/adding-credentials-to-jetpack/"
+					>
+						{ translate( 'Need help? Find your server credentials' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Need help finding your site's server credentials?' )` *ES Score: 9*